src/components/customerCases/customerCase/sections/results/ResultsBlock.tsx

```diff
@@ -5,23 +5,34 @@ import styles from "./resultsBlock.module.css";
 
 export interface ResultsBlockProps {
   section: ResultsBlockObject;
+  blockColor?: string;
 }
 
-export default function ResultsBlock({ section }: ResultsBlockProps) {
+export default function ResultsBlock({
+  section,
+  blockColor,
+}: ResultsBlockProps) {
   return (
     section.resultsBlockTitle && (
       <div className={styles.wrapper}>
         <div>
-          <StackedHighlights section={section}></StackedHighlights>
+          <StackedHighlights
+            section={section}
+            blockColor={blockColor}
+          ></StackedHighlights>
```

Comment:
```suggestion
          <StackedHighlights
            section={section}
            blockColor={blockColor}
          />
```
I think you can simplify like this 😊